index.html

```diff
@@ -6,6 +6,392 @@
 	<link rel="stylesheet" type="text/css" href="index.css">
 </head>
 <body>
-	
+
+	<!-- intro -->
+	<header class="row">
+
+		<img src="assets/yellow-pill.svg" alt="yellow pill">
```

Comment:
Yes, @benjaminmodayil is right here.  Only images which are real content should have alt text.  Images which are for design only should be `alt=""`, so that a screen reader ignores them entirely.